packages/visx-zoom/test/Zoom.test.tsx

```diff
@@ -33,8 +33,7 @@ describe('<Zoom />', () => {
       </Zoom>,
     );
 
-    expect(wrapper.find('div')).toHaveLength(1);
```

Comment:
I think previously when `passive=false` (default), `Zoom` wrapped `children` in a `div` so that's why this is no longer there 👍